packages/dashboards/src/lib/components/timeseries-widget/chart-presets/status-bar-chart/status-bar-chart.component.ts

```diff
@@ -297,6 +297,19 @@ export class StatusBarChartComponent
                 configuration.gridConfig.sideMargin;
         }
     }
+
+    public displayDeleteButton(): boolean {
+        return !!this.configuration.allowDelete;
```

Comment:
on what allowDelete depends?

could you use configuration.allowLegendMenu?

please also add a check for TimeseriesWidgetProjectType.PerfstackApp
https://github.com/solarwinds/nova/pull/553/files